sources/platform/runtime/machine-image/ami-overview.md

```diff
@@ -40,33 +40,29 @@ You can click on any image below to see what is pre-installed:
 | [v5.3.2](ami-v532/)        | Mar 11, 2017    |
 | [Stable](ami-stable/)      | Feb 19, 2016 (deprecated) |
 
-In most cases, the default Machine image set for your Subscription will be sufficient to run your builds. The main reasons why you might want to consider
-changing to a more recent image are:
+In most cases, the default Machine image set for your Subscription will be sufficient to run your builds. The main reasons why you might want to consider changing to a more recent image are:
 
 -  You need a newer language/service/package version
 -  You need a newer Docker version
 
 <a name="view-machine-image"></a>
-## Viewing Subscription Machine Image
 
-To see what Machine Image is being used for your subscription:
+## Viewing Subscription Machine Images
 
-* From the left sidebar menu, click on your Subscription
+When a subscription is imported in your Shippable account for the very first time, a [Node pool](/platform/management/subscription/node-pools/) is automatically created for you. The default Node pool has a single Ubuntu 16.04 on-demand node, with the latest released Ubuntu 16.04 machine image. A subscription can also have multiple Node pools that you can create from the billing page. Each node pool specifies the machine image in its `Runtime Version` setting.
 
-<img src="/images/getting-started/account-settings.png" alt="View Subscription">
+Here are the steps to view the `Runtime Version` setting:
 
-* On your Subscription Dashboard, click on **Settings**, i.e. the **Gears** icon, on the top right which will show a dropdown. Click on **Nodes**
+* View your subscription Node pool using steps documented [here](/platform/management/subscription/node-pools/#viewing-subscription-node-pools)
+* Click on `Edit` on your Node pool to view the `Runtime Version`.
 
-<img src="/images/ci/view-machine-image.png"
-alt="Machine Image for a Subscription" style="width:800px;"/>
+<img src="/images/platform/management/subscription-node-pools-edit.png" alt="Subscription Node Pools view for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
 
-<a name="change-machine-image"></a>
 ## Changing the Subscription Machine image
 
-To view your machine, image follow instructions in [Viewing Subscription Machine Image](#view-machine-image) section above.
+* Follow instructions in the [Viewing Subscription Machine Image](#view-machine-image) section above to view the Node pool for your subscription.
 
-You can click on the **Machine Images** dropdown to select a different image. Please note that this setting will
-affect all projects and builds in your Subscription.
+* Click on the **Runtime Version** dropdown to select a different image version. Please note that this setting will
+affect all projects and builds that use this Node pool.
```

Comment:
this will only affect the builds that are running on the node pool. the builds can be forced to run on a node pool by changing `runtime` setting in their job yml